### PR TITLE
Fixed Liquid properties documentation

### PR DESCRIPTION
**Pull request by PiemP**
Fixed the name of editor template in ContentFields docs
Fixed Bootstrap template for NumericField editor: added a div for col-sm-3 element

### CODE REVIEW

**Comment by sebastienros** [comment]
Can you share a screenshot for before and after the change?

**Comment by PiemP** [comment]
After:
4_after
Before:
4_before

**Comment by Skrypt** [comment]
Can you fix it for the "indicator" one too? It's not getting the same width there in your screenshot.

**Comment by PiemP** [comment]
I have removed the jquery UI spinner code because seems bootstrap/HTML5 work well with input field with attribute type set as a number.

**Comment by PiemP** [comment]
I have added a little fix to the documentations about shape_add_properties Liquid filter

**Comment by Skrypt** [comment]
Please fix conflicts so that I can review.

**Comment by sebastienros** [comment]
As @Skrypt is asking, why is the second editor in your screenshots not aligned?

**Comment by PiemP** [comment]
numeric
Sorry, this is the new screenshot.

**Comment by agriffard** [comment]
Please solve the conflicts.
Please try to make different PRs when you want to fix separate issues.

File: src/docs/reference/modules/Liquid/README.md [hunk 1]
```diff
@@ -603,7 +603,7 @@ Input
 
 Adds properties to a shape. This can be useful to pass values from a parent shape.  
 Property names get converted to PascalCase.  
-Ex: `prop_name1` can be accessed via `Model.PropName1` in the shape template.
+Ex: `prop_name1` can be accessed via `Model.Properties["PropName1"]` or `Model.Properties.PropName1` in the shape template.
```
Comment (Skrypt) [inline src/docs/reference/modules/Liquid/README.md:606]: We can keep this. The remaining has been fixed by @netwavebe

**Review by Skrypt — CHANGES_REQUESTED** [review]

File: src/OrchardCore.Modules/OrchardCore.ContentFields/Views/NumericField-Slider.Edit.cshtml [hunk 2]
```diff
@@ -39,9 +39,10 @@
     }
 </div>
 <script at="Foot">
-    $('#@(id)-slider').bootstrapSlider({ tooltip: 'always' }).on('change', function (ev) {
-        $('#@(id)').val(ev.value.newValue);
+    $('#@(id)-slider').bootstrapSlider({ tooltip: 'always' }).on('slide', function (ev) {
```
Comment (Skrypt) [inline src/OrchardCore.Modules/OrchardCore.ContentFields/Views/NumericField-Slider.Edit.cshtml:42]: Revert this change, keep what we have

File: src/OrchardCore.Modules/OrchardCore.ContentFields/Views/NumericField-Spinner.Edit.cshtml [hunk 3]
```diff
@@ -45,4 +45,4 @@
             step: @step
         });
     });
-</script>
+</script>
```
Comment (Skrypt) [inline src/OrchardCore.Modules/OrchardCore.ContentFields/Views/NumericField-Spinner.Edit.cshtml:48]: eof

File: src/OrchardCore.Modules/OrchardCore.ContentFields/Views/NumericField-Slider.Edit.cshtml (hunk 2 above)
Comment (netwavebe) [reply src/OrchardCore.Modules/OrchardCore.ContentFields/Views/NumericField-Slider.Edit.cshtml:42]: Yes, keep 'change' event because 'slide' won't show values while dragging the slider.

Comment (Skrypt) [reply src/OrchardCore.Modules/OrchardCore.ContentFields/Views/NumericField-Slider.Edit.cshtml:42]: And I'm guessing that ev.value.newValue is necessary too here.

Comment (netwavebe) [reply src/OrchardCore.Modules/OrchardCore.ContentFields/Views/NumericField-Slider.Edit.cshtml:42]: Yes, ev.value is an object that holds both the old and new value in this case.
